local_archives/nwp/gfs/gfs.py

```diff
@@ -42,3 +55,4 @@ def zarr_archive(
             },
         },
     ).get_materialize_result()
+    """
```

Comment:
im not sure about these """, but you might have more context on how this all fits in

Reply:
Fair point - was going to add it back in later but I can just refer to the PR instead.